core/store/src/main/java/uk/gov/gchq/gaffer/store/SchemaOperationChainValidator.java

```diff
@@ -28,12 +27,7 @@
 
     Schema schema;
 
-    public SchemaOperationChainValidator(final ViewValidator viewValidator) {
```

Comment:
For the sake of backwards compatibility, can you deprecate the old constructors rather than removing them?